scripts/mbedtls_dev/bignum_common.py

```diff
@@ -287,6 +302,8 @@ def r2(self) -> int: # pylint: disable=invalid-name
     def is_valid(self) -> bool:
         if self.int_a >= self.int_n:
             return False
+        if self.disallow_zero_a and self.int_a == 0:
```

Comment:
When I did this, I got errors from pylint about too much duplicated code.

This is an ongoing problem, and I think we're a bit close to deadline to change what we're doing now, which is: to avoid duplication, if code needs to be repeated enough for pylint to complain, it needs to be an option in the base class.

I don't like this, but I don't like Python either :) and nor do I like that our code-checking tools are preventing us writing the code we would like to write. This is the tail wagging the dog! but it's the route we've chosen to go down.